observation_portal/requestgroups/duration_utils.py

```diff
@@ -58,19 +58,21 @@ def get_configuration_duration(configuration_dict):
         conf_duration['duration'] = configuration_dict['repeat_duration']
     else:
         conf_duration['duration'] = sum([icd['duration'] for icd in instrumentconf_durations])
-    conf_duration['duration'] += (PER_CONFIGURATION_STARTUP_TIME)
+        conf_duration['duration'] += request_overheads['config_front_padding']
     return conf_duration
 
 
 def get_request_duration_dict(request_dict, is_staff=False):
     req_durations = {'requests': []}
     for req in request_dict:
         req_info = {'duration': get_request_duration(req)}
-        conf_durations = [get_configuration_duration(conf) for conf in req['configurations']]
+        conf_durations = []
+        for conf in req['configurations']:
+            request_overheads = configdb.get_request_overheads(conf['instrument_type'])
+            conf_durations.append(get_configuration_duration(conf, request_overheads))
         req_info['configurations'] = conf_durations
         rise_set_intervals = get_filtered_rise_set_intervals_by_site(req, is_staff=is_staff)
         req_info['largest_interval'] = get_largest_interval(rise_set_intervals).total_seconds()
-        req_info['largest_interval'] -= (PER_CONFIGURATION_STARTUP_TIME)
```

Comment:
This "largest_interval" is used for computing repeat_expose / fill_window stuff. By removing this, we are making the largest_interval larger and implicitly including the per configuration_front_padding into the repeat_duration that will get set, rather than adding it later.

Reply:
I think that sounds good